Patch build.rs to output the source distribution conditions of the library.

--- a/build.rs	2019-06-03 14:17:18.866924455 +0200
+++ b/build.rs	2019-06-03 14:17:49.042557720 +0200
@@ -1,6 +1,10 @@
 extern crate cc;
 
+use std::env;
+
 fn main() {
+    println!("dh-cargo:deb-built-using=ansi256=1={}", env::var("CARGO_MANIFEST_DIR").unwrap());
+
     cc::Build::new()
         .file("src/ansi256.c")
         .compile("ansi256");
